gslab_scons/configuration_tests.py

```diff
@@ -132,10 +132,10 @@ def check_stata(packages = ["yaml"], user_yaml = "user-config.yaml"):
     Check that a valid Stata executable is in the path and that the specified
     Stata packages are installed.
     '''
-    flavor = load_yaml_value(user_yaml, "stata_executable")
+    executable = load_yaml_value(user_yaml, "stata_executable")
 
     # Fake scons-like env dict for misc.get_stata_executable(env)
-    fake_env = {'user_flavor': flavor} 
+    fake_env = {'stata_executable': executable} 
     stata_exec = get_stata_executable(fake_env)
```

Comment:
Minor, but we refer to this object as `stata_executable` elsewhere. It might be worth a change.